src/pages/iou/IOUCurrencySelection.js

```diff
@@ -88,7 +88,7 @@ class IOUCurrencySelection extends Component {
      */
     changeSearchValue(searchValue) {
         const currencyOptions = this.getCurrencyOptions(this.props.currencyList);
-        const searchRegex = new RegExp(searchValue, 'i');
+        const searchRegex = new RegExp(searchValue.replace(/[.*+?^${}()|[\]\\]/g, '\\$&'), 'i');
```

Comment:
@iwiznia, correct me if I'm wrong, but can't we use `Str.escapeForRegExp(searchValue)` here instead? I've tested and it seems to work fine, but maybe I'm just missing and edge-case you've encountered

Reply:
Oh! I did not know we had that. I think they are the same, but the other one escapes `/` and `-`, which I _think_ is correct. Changing to that one.